### ISSUE TITLE
First test of Komodo extension

### ISSUE THREAD

**Issue by Desiderius77** [zeelow/custom4EAST#1]
I have just tested the first implementation of the EAST language in Komodo : 

- Good starting point !

I have noticed some pitfalls : 

- All the EAST elements appear when typing the first letters but they do no seem to be understood as XML elements . For instance if I Type "EA" I see a "block" " EAST" appearing
but if I click on Enter only yhe word EAST is completed without closing the beginning tag and without showing the corresponding closing tag as Komodo does usally with XML elements.

- Same behavior with the attrbutes which are considered as character entities

- The XML schema is not dynamically verified ; we can insert elements which are not allowed in the given context ( is it possible with Komodo ?? )

**Comment by hamlety** [comment]
For the moment, all elements and attributes are considering like keyword, just that. The auto-close of tags is not implement, tutorial or comments are really rare on this subject.
On the verification I don't find any information about this, for the moment.

**Comment by Desiderius77** [comment]
Perhaps look at the XML part of the documentation.
If you open a new document and declare it as an XML document, you will see every time you
open a tag, the closing tag is written and event an identation is set.
Do no hesitate to ask questions on the mailing lists and/or on the IRC channel

**Comment by hamlety** [comment]
Yes all XML document work on XML like this. All the stuff I doing don't match with XML, it's configure to be an XML extension but nothing change. I must create the fonction to work like this but it failled each time, so I work on.
